migrations/versions/ce359a457d3d_.py

```diff
@@ -0,0 +1,43 @@
+"""
```

Comment:
Did you try this on a db with data? Want to make sure that the existing constraints don't break with the name changes.